components/UpcomingEvents.js

```diff
@@ -64,7 +64,7 @@ const UpcomingEvents = ({
             return (
               <div
                 key={ event.get('_id') }
-                className={`event-preview relative ${list?'':'pr-4 md:w-1/3 mb-8'}`}
+                className={`event-preview relative ${list?'':'pr-4 md:w-12/12 mb-8'}`}
```

Comment:
This breaks the event page. In order to enable full widths event you should just pass the `list` prop.